workshops/models.py

```diff
@@ -227,6 +227,16 @@ def unpublished_events(self):
 
         return self.filter(published=False)
 
+    def unpaid_events(self):
+        '''Return a queryset for events that owe money.'''
+
+        # All events that have a non-zero admin fee.
+        queryset = self.filter(admin_fee__ne=0)
+
+        # Of those, all those whose paid status is not 'Yes' (i.e., is 'No' or NULL).
+        queryset = queryset.filter(fee_paid__ne=True)
```

Comment:
On Wed, May 06, 2015 at 12:44:45PM -0700, Piotr Banaszkiewicz wrote:
>         # All events that have a non-zero admin fee.
>         queryset = self.filter(Q(admin_fee__gt=0) | Q(admin_fee__lt=0))

What is the < 0 case for?

>         # Of those, all those whose paid status is not 'Yes' (i.e., is 'No' or NULL).
>         queryset = queryset.filter(~Q(fee_paid=True))

No need to split this into multiple filter calls.  How about:

  self.filter(admin_fee__gt=0).exclude(fee_paid=True)

?